src/actions/dataActions.tsx

```diff
@@ -142,10 +142,18 @@ export function shareStatePermalink() {
           const location = window.location;
           const viewerUrl = location.origin + location.pathname;
           const stateUrl = `${viewerUrl}?stateKey=${stateKey}`;
+
+          const expiration = Config.instance.branding.permalinkExpirationDays;
+          const message =
+            expiration !== undefined
```

Comment:
This is more robust in JS/TS:

```suggestion
            typeof expiration === "number"
```

Reply:
Great improvement. Thanks for the suggestion!